app/models/user.rb

```diff
@@ -328,6 +328,13 @@ def destroy_empty_series
       destroy_all
   end
 
+  def assignments
+    offers = ChallengeAssignment.joins(offer_signup: :pseud).where(pseuds: { user_id: self.id })
+    pinch_hits = ChallengeAssignment.joins(:pinch_hitter).where(pseuds: { user_id: self.id })
```

Comment:
Is there a reason we want to avoid using `offer_assignments` and `pinch_hit_assignments` here? They already only include sent assignments.